### DISCUSSION TITLE
Combining Freeplane and Thunderbird for mindmap view and calendar view of same data

### DISCUSSION THREAD

**Discussion by sgjohnson1981** [freeplane/freeplane#821] [Ideas]
What would be the easiest way to connect Freeplane and Thunderbird's "Lightning" calendar interface, or some other free calendar/task software? I've been using Freeplane for a long time and would love some way of seeing some of my nodes in a calendar/task view with some easy drag/drop interface like Thunderbird, i.e. move things around in Thunderbird calendar and the changes are reflected in my mindmap. Or does anyone have a better way of todo type stuff already available in Freeplane? I have zero development experience and little programming experience, but would help out any way possible anyway.

**Comment by bepolymathe** [comment]
Hi, 

It's true that it would be ideal to be able to manage your agenda from freeplane in connection with a shareable calendar. I imagine that it would be possible to imagine an addon allowing to enter as attribute the elements of an event which would be then exported as .ics 🤔
